alignak_backend_client/backend_client.py

```diff
@@ -811,7 +811,8 @@ def create_update_resource(self, resource_name, name, update=False):
             count = 0
             for json_item in json_data:
                 logger.info("-> json item: %s", json_item)
-                if name is None and ('name' not in json_item or not json_item['name']):
+                if resource_name not in ['history'] and name is None \
```

Comment:
You have too the ressources `userrestrictrole`, `logcheckresult`, `alignakretention`, `actionforcecheck`, `actiondowntime` and `actionacknowledge` that not have `name` field

Reply:
No need to import such resource with this utility

Reply:
Perharps userrestrictrole indeed... I will improve this